libcodechecker/cmd/cmd_line_client.py

```diff
@@ -441,6 +496,7 @@ def print_reports(client, reports, output_format):
                     get_line_from_file(report.main['location']['file_name'],
                                        bug_line)
             else:
```

Comment:
Do we know the type that comes from the server? Maybe it should be explicitly put as an `elif`, instead of an `else`.